include/GoogleSync/GoogleSync.php

```diff
@@ -634,39 +755,28 @@ public function pushEvent($event_local, $event_remote = null)
         $this->setLastSync($event_local, $return->getId());
```

Comment:
setLastSync() return value could be false as failure. Please handle the error.

Reply:
Done